diff -ru rnammer-1.2.orig/core-rnammer rnammer-1.2/core-rnammer
--- rnammer-1.2.orig/core-rnammer	2020-09-11 11:12:35.645289000 -0500
+++ rnammer-1.2/core-rnammer	2020-09-11 11:22:25.075553611 -0500
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#!/usr/bin/env perl
 use Data::Dumper;
 # VERSION 1.2
 my $version = '1.2-core';
@@ -111,7 +111,7 @@
 				push @OPTIONS , "--domT $CONFIG{domain_spottermodel_score}" unless $CONFIG{domain_spottermodel_score} eq '';
 				push @OPTIONS , "-E $CONFIG{global_spottermodel_evalue}" unless $CONFIG{global_spottermodel_evalue} eq '';
 				push @OPTIONS , "-T $CONFIG{global_spottermodel_score}" unless $CONFIG{global_spottermodel_score} eq '';
-				system sprintf('%s --cpu 1 --compat %s %s "%s" > "%s.hmmsearchresult"',$CONFIG{hmmsearch},join(' ',@OPTIONS),$CONFIG{spottermodel},$fname,$fname);
+				system sprintf('%s --compat %s %s "%s" > "%s.hmmsearchresult"',$CONFIG{hmmsearch},join(' ',@OPTIONS),$CONFIG{spottermodel},$fname,$fname);
 			}
 		}
 		while (wait != -1) { sleep 1; }
@@ -184,7 +184,7 @@
 		$jobs{$subfasta_key}{fname} = $fname;
 		push @tempfileS, $fname;
 		push @tempfileS, "$fname.hmmsearchresult";
-		system sprintf('%s --cpu 1 --compat %s "%s" "%s" > "%s.hmmsearchresult"',$CONFIG{hmmsearch},join(' ',@OPTIONS),$CONFIG{postmodel},$fname,$fname);
+		system sprintf('%s --compat %s "%s" "%s" > "%s.hmmsearchresult"',$CONFIG{hmmsearch},join(' ',@OPTIONS),$CONFIG{postmodel},$fname,$fname);
 	}
 	while (wait != -1) { sleep 1; }
 	if ($CONFIG{hmm_output} ne '') {
diff -ru rnammer-1.2.orig/rnammer rnammer-1.2/rnammer
--- rnammer-1.2.orig/rnammer	2020-09-11 11:12:35.616769000 -0500
+++ rnammer-1.2/rnammer	2020-09-11 11:19:35.058125511 -0500
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#!/usr/bin/env perl
 # VERSION 1.2
 my $version = '1.2';
 # 1.1b: config parameter 'multi' introduced to allow parallel execution
@@ -32,7 +32,7 @@
 ## PROGRAM CONFIGURATION BEGIN
 
 # the path of the program
-my $INSTALL_PATH = "/usr/cbs/bio/src/rnammer-1.2";
+my $INSTALL_PATH = "/home/apps/software/RNAmmer/1.2-IGB-gcc-4.9.4-Perl-5.24.1";
 
 # The library in which HMMs can be found
 my $HMM_LIBRARY = "$INSTALL_PATH/lib";
@@ -47,8 +47,9 @@
 my $HMMSEARCH_BINARY;
 my $PERL;
 if ( $uname eq "Linux" ) {
-	$HMMSEARCH_BINARY = "/usr/cbs/bio/bin/linux64/hmmsearch";
-	$PERL = "/usr/bin/perl";
+	#$HMMSEARCH_BINARY = "/usr/cbs/bio/bin/linux64/hmmsearch";
+	$HMMSEARCH_BINARY = `which hmmsearch2`;
+	$PERL = "perl";
 } elsif ( $uname eq "IRIX64" ) {
 	$HMMSEARCH_BINARY = "/usr/cbs/bio/bin/irix64/hmmsearch";
 	$PERL = "/usr/sbin/perl";
@@ -179,7 +180,7 @@
 	push  @TEMP_FILES , $TEMP_CF_FILE unless $keep;
 	if ($multi) {
 		unless (fork) {
-			open CORE , "$PERL $RNAMMER_CORE $TEMP_CF_FILE |"; 
+			open CORE , "$RNAMMER_CORE $TEMP_CF_FILE |"; 
 			while (<CORE>) {
 				$faultstring .= $_;
 			}
@@ -188,7 +189,7 @@
 			exit;
 		}
 	} else {
-			open CORE , "$PERL $RNAMMER_CORE $TEMP_CF_FILE |"; 
+			open CORE , "$RNAMMER_CORE $TEMP_CF_FILE |"; 
 			while (<CORE>) {
 				$faultstring .= $_;
 			}
diff -ru rnammer-1.2.orig/xml2fsa rnammer-1.2/xml2fsa
--- rnammer-1.2.orig/xml2fsa	2020-09-11 11:12:35.646338000 -0500
+++ rnammer-1.2/xml2fsa	2020-09-11 11:17:05.035256000 -0500
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#!/usr/bin/env perl
 # Peter Hallin 2007-02-06, to be used with RNAmmer v1.2
 
 use XML::Simple;
diff -ru rnammer-1.2.orig/xml2gff rnammer-1.2/xml2gff
--- rnammer-1.2.orig/xml2gff	2020-09-11 11:12:35.644213000 -0500
+++ rnammer-1.2/xml2gff	2020-09-11 11:17:05.037469000 -0500
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#!/usr/bin/env perl
 # Peter Hallin 2007-02-06, to be used with RNAmmer v1.2
 
 use XML::Simple;
