diff -ru --new-file homer-4.9.1.orig/bin/addGeneAnnotation.pl homer-4.9.1/bin/addGeneAnnotation.pl
--- homer-4.9.1.orig/bin/addGeneAnnotation.pl	2016-07-05 13:15:17.000000000 -0500
+++ homer-4.9.1/bin/addGeneAnnotation.pl	2017-11-07 15:50:52.399885335 -0600
@@ -55,9 +55,9 @@
 
 `convertIDs.pl "$file" $organism gene $header yes $keep > $tmpfile`;
 if ($header eq 'yes') {
-	`addDataHeader.pl $tmpfile "$homeDir/data/accession/$organism.description" > $tmpfile2`;
+	`addDataHeader.pl $tmpfile "$ENV{'HOMER_DATA'}/data/accession/$organism.description" > $tmpfile2`;
 } else {
-	`addData.pl $tmpfile "$homeDir/data/accession/$organism.description" > $tmpfile2`;
+	`addData.pl $tmpfile "$ENV{'HOMER_DATA'}/data/accession/$organism.description" > $tmpfile2`;
 }
 
 open IN, $tmpfile2;
diff -ru --new-file homer-4.9.1.orig/bin/analyzeRNA.pl homer-4.9.1/bin/analyzeRNA.pl
--- homer-4.9.1.orig/bin/analyzeRNA.pl	2016-07-05 13:15:17.000000000 -0500
+++ homer-4.9.1/bin/analyzeRNA.pl	2017-11-07 15:51:09.010926650 -0600
@@ -744,7 +744,7 @@
 		return;
 	}
 
-	my $convFile = $homeDir . "/data/accession/$organism" . "2gene.tsv";
+	my $convFile = "$ENV{'HOMER_DATA'}/data/accession/$organism" . "2gene.tsv";
 	unless (-f $convFile) {
 		return;
 	}
diff -ru --new-file homer-4.9.1.orig/bin/annotatePeaks.pl homer-4.9.1/bin/annotatePeaks.pl
--- homer-4.9.1.orig/bin/annotatePeaks.pl	2017-02-22 12:39:29.000000000 -0600
+++ homer-4.9.1/bin/annotatePeaks.pl	2017-11-07 15:51:22.551960321 -0600
@@ -3673,7 +3673,7 @@
 
 
 my %ug2gene = ();
-my $convFile = $homeDir . "/data/accession/$organism" . "2gene.tsv";
+my $convFile = "$ENV{'HOMER_DATA'}/data/accession/$organism" . "2gene.tsv";
 my @geneData = ();
 my @geneDataHeader = (); 
 my %acc2gene = ();
@@ -3692,7 +3692,7 @@
 	}
 	close IN;
 
-	my $descriptionFile = $homeDir . "/data/accession/$organism.description";
+	my $descriptionFile = "$ENV{'HOMER_DATA'}/data/accession/$organism.description";
 	open IN, $descriptionFile;
 	while (<IN>) {
 		chomp;
diff -ru --new-file homer-4.9.1.orig/bin/checkTagBias.pl homer-4.9.1/bin/checkTagBias.pl
--- homer-4.9.1.orig/bin/checkTagBias.pl	2016-07-05 13:15:17.000000000 -0500
+++ homer-4.9.1/bin/checkTagBias.pl	2017-11-07 15:51:42.775010597 -0600
@@ -21,7 +21,7 @@
 use POSIX;
 use HomerConfig;
 
-my $knownMotifs = $homeDir . "/data/knownTFs/JASPAR.matrix";
+my $knownMotifs = "$ENV{'HOMER_DATA'}/data/knownTFs/JASPAR.matrix";
 
 my $config = HomerConfig::loadConfigFile();
 $outputFileName = "tagFreq.txt";
diff -ru --new-file homer-4.9.1.orig/bin/compareMotifs.pl homer-4.9.1/bin/compareMotifs.pl
--- homer-4.9.1.orig/bin/compareMotifs.pl	2017-02-20 16:19:26.000000000 -0600
+++ homer-4.9.1/bin/compareMotifs.pl	2017-11-07 15:51:55.879043168 -0600
@@ -11,8 +11,8 @@
 
 use HomerSVGLogo;
 
-my $motifDir = $homeDir . "/data/knownTFs/";
-my $chuckFacts = $homeDir . "/data/misc/ChuckFacts.tsv";
+my $motifDir = "$ENV{'HOMER_DATA'}/data/knownTFs/";
+my $chuckFacts = "$ENV{'HOMER_DATA'}/data/misc/ChuckFacts.tsv";
 
 my $fpThresh = 1e-12;
 $TopN = 10;
diff -ru --new-file homer-4.9.1.orig/bin/convertIDs.pl homer-4.9.1/bin/convertIDs.pl
--- homer-4.9.1.orig/bin/convertIDs.pl	2016-07-05 13:15:16.000000000 -0500
+++ homer-4.9.1/bin/convertIDs.pl	2017-11-07 15:52:09.715077551 -0600
@@ -4,7 +4,7 @@
 my $homeDir = "/gpfs/data01/cbenner/software/homer/./";
 
 my $promoterSeqOffset = -2000;
-my $accDir = $homeDir . "/data/accession/";
+my $accDir = "$ENV{'HOMER_DATA'}/data/accession/";
 
 if (@ARGV < 3) {
 	print STDERR "\n\tUsage: convertIDs.pl <input file> <organism> <ID-type> [header] [keep original] [keep all]\n";
diff -ru --new-file homer-4.9.1.orig/bin/convertOrganismID.pl homer-4.9.1/bin/convertOrganismID.pl
--- homer-4.9.1.orig/bin/convertOrganismID.pl	2016-07-05 13:15:17.000000000 -0500
+++ homer-4.9.1/bin/convertOrganismID.pl	2017-11-07 15:52:25.666117182 -0600
@@ -4,11 +4,11 @@
 my $homeDir = "/gpfs/data01/cbenner/software/homer/./";
 
 
-my $homologeneFile = $homeDir . "/data/accession/homologene.data";
-my $taxFile = $homeDir . "/data/accession/taxids.tsv";
+my $homologeneFile = "$ENV{'HOMER_DATA'}/data/accession/homologene.data";
+my $taxFile = "$ENV{'HOMER_DATA'}/data/accession/taxids.tsv";
 
 my %taxid = ();
-open IN, $taxFile or die "!! Could not find the taxonomy ID file in $homeDir/data/accession/taxid.tsv\nMight need to update the software\n";
+open IN, $taxFile or die "!! Could not find the taxonomy ID file in $ENV{'HOMER_DATA'}/data/accession/taxid.tsv\nMight need to update the software\n";
 while (<IN>) {
 	chomp;
 	s/\r//g;
diff -ru --new-file homer-4.9.1.orig/bin/findGO.pl homer-4.9.1/bin/findGO.pl
--- homer-4.9.1.orig/bin/findGO.pl	2016-07-22 19:37:20.000000000 -0500
+++ homer-4.9.1/bin/findGO.pl	2017-11-07 15:52:43.591161707 -0600
@@ -4,10 +4,10 @@
 my $homeDir = "/gpfs/data01/cbenner/software/homer/./";
 my $promoterSeqOffset = -2000;
 
-my $goDir = $homeDir . "/data/GO/";
-my $goFile = $homeDir . "/data/GO/GO.txt";
-my $promoterDir = $homeDir . "/data/promoters/";
-my $accDir = $homeDir . "/data/accession/";
+my $goDir = "$ENV{'HOMER_DATA'}/data/GO/";
+my $goFile = "$ENV{'HOMER_DATA'}/data/GO/GO.txt";
+my $promoterDir = "$ENV{'HOMER_DATA'}/data/promoters/";
+my $accDir = "$ENV{'HOMER_DATA'}/data/accession/";
 my $pvalueThresh = 0.05;
 
 
diff -ru --new-file homer-4.9.1.orig/bin/findKnownMotifs.pl homer-4.9.1/bin/findKnownMotifs.pl
--- homer-4.9.1.orig/bin/findKnownMotifs.pl	2017-02-20 16:13:01.000000000 -0600
+++ homer-4.9.1/bin/findKnownMotifs.pl	2017-11-07 15:52:53.309185842 -0600
@@ -11,7 +11,7 @@
 
 my $analysisType = "GETPVALUE";
 
-my $knownFile  = $homeDir . "/data/knownTFs/known.motifs";
+my $knownFile  = "$ENV{'HOMER_DATA'}/data/knownTFs/known.motifs";
 my $pvalueThresh = 0.01;
 my $cpus = 1;
 my $cache = 500;
diff -ru --new-file homer-4.9.1.orig/bin/findMotifsGenome.pl homer-4.9.1/bin/findMotifsGenome.pl
--- homer-4.9.1.orig/bin/findMotifsGenome.pl	2017-02-20 19:44:06.000000000 -0600
+++ homer-4.9.1/bin/findMotifsGenome.pl	2017-10-17 13:27:05.687274693 -0500
@@ -265,8 +265,8 @@
 			$cmd->{'noknown'} = 1; 
 			$cmd->{'rnaMode'} = 1; 
 			$cmd->{'norevopp'} = 1; 
-			$cmd->{'mknown'} = $homeDir . "/data/knownTFs/known.rna.motifs";
-			$cmd->{'mcheck'} = $homeDir . "/data/knownTFs/all.rna.motifs";
+			$cmd->{'mknown'} = $ENV{'HOMER_DATA'} . "/knownTFs/known.rna.motifs";
+			$cmd->{'mcheck'} = $ENV{'HOMER_DATA'} . "/knownTFs/all.rna.motifs";
 			print STDERR "\tOperating in RNA mode\n";
 		} elsif ($ARGV[$i] eq '-reuse' ) {
 			$cmd->{'reuse'} = 1; 
diff -ru --new-file homer-4.9.1.orig/bin/findMotifs.pl homer-4.9.1/bin/findMotifs.pl
--- homer-4.9.1.orig/bin/findMotifs.pl	2017-02-20 19:46:56.000000000 -0600
+++ homer-4.9.1/bin/findMotifs.pl	2017-11-07 15:53:14.851239331 -0600
@@ -24,8 +24,8 @@
 use Statistics;
 
 #default options...
-my $accDir = $homeDir . "/data/accession/";
-my $seqDir = $homeDir . "/data/promoters/";
+my $accDir = $ENV{'HOMER_DATA'} . "/data/accession/";
+my $seqDir = $ENV{'HOMER_DATA'} . "/data/promoters/";
 my $reduceThresh = 0.6;
 my $matchThresh = "T10";
 my $knownPvalueThresh = 0.01;
@@ -265,8 +265,8 @@
 			$cmd->{'rnaMode'} = 1;
 			$cmd->{'norevopp'} = 1;
 			$cmd->{'noknown'} = 1;
-			$cmd->{'mknown'} = $homeDir . "/data/knownTFs/known.rna.motifs";
-			$cmd->{'mcheck'} = $homeDir . "/data/knownTFs/all.rna.motifs";
+			$cmd->{'mknown'} = $ENV{'HOMER_DATA'} . "/data/knownTFs/known.rna.motifs";
+			$cmd->{'mcheck'} = $ENV{'HOMER_DATA'} . "/data/knownTFs/all.rna.motifs";
 		} elsif ($ARGV[$i] eq '-opt') {
 			my $bail = 0;
 			print STDERR "\tWill optimize motifs in the following files:\n";
diff -ru --new-file homer-4.9.1.orig/bin/GenomeOntology.pl homer-4.9.1/bin/GenomeOntology.pl
--- homer-4.9.1.orig/bin/GenomeOntology.pl	2016-07-05 13:15:17.000000000 -0500
+++ homer-4.9.1/bin/GenomeOntology.pl	2017-11-07 15:53:24.859264175 -0600
@@ -34,7 +34,7 @@
 	print STDERR "\n\tUsage: GenomeOntology.pl <peak file/Tag Directory> <genome> <Output Directory> [additional options]\n";
 	print STDERR "\n\tThis produces an HTML page and additional data files in the <Output Directory>.  \n";
 	print STDERR "\tTo customize this analysis, place additional peak/annotation files in the directory:\n";
-	print STDERR "\t\t$homeDir/data/genomes/\"GENOME\"/annotations/custom/\n";
+	print STDERR "\t\t$ENV{'HOMER_DATA'}/data/genomes/\"GENOME\"/annotations/custom/\n";
 
 	print STDERR "\n\tPossible Genomes:\n";
 	foreach(keys %{$config->{'GENOMES'}}) {
diff -ru --new-file homer-4.9.1.orig/bin/getDistalPeaks.pl homer-4.9.1/bin/getDistalPeaks.pl
--- homer-4.9.1.orig/bin/getDistalPeaks.pl	2016-07-05 13:15:17.000000000 -0500
+++ homer-4.9.1/bin/getDistalPeaks.pl	2017-11-07 15:53:34.267287527 -0600
@@ -47,7 +47,7 @@
 	} elsif ($ARGV[$i] eq '-d') {
 		$distance = $ARGV[++$i];
 	} elsif ($ARGV[$i] eq '-noTTS') {
-		$ttsFlag  = " -pdist -p \"$homeDir/data/genomes/$genome/$genome.tts\" ";
+		$ttsFlag  = " -pdist -p \"$ENV{'HOMER_DATA'}/data/genomes/$genome/$genome.tts\" ";
 	} elsif ($ARGV[$i] eq '-targets') {
 		$targetFlag = 1;
 	} else {
diff -ru --new-file homer-4.9.1.orig/bin/HomerConfig.pm homer-4.9.1/bin/HomerConfig.pm
--- homer-4.9.1.orig/bin/HomerConfig.pm	2017-02-20 10:03:23.000000000 -0600
+++ homer-4.9.1/bin/HomerConfig.pm	2017-11-07 15:54:06.059366417 -0600
@@ -11,7 +11,7 @@
 
 	my ($file) = @_;
 	if (!defined($file)) {
-		$file = $homeDir . "/config.txt";
+		$file = $ENV{'HOMER_DATA'} . "/config.txt";
 	}
 	my %a = ();
 	my %b = ();
@@ -31,7 +31,7 @@
 sub printConfigFile {
 	my ($config,$outFile, $updateFlag) = @_;
 
-	my $file = $homeDir . "/config.txt";
+	my $file = $ENV{'HOMER_DATA'} . "/config.txt";
 	if (defined($outFile)) {
 		$file = $outFile;
 		#print STDERR "\tPrinting Configuration to $outFile (may not be standard...)\n";
@@ -356,7 +356,7 @@
 	my ($mset, $org) = @_;
 	if ($mset eq 'auto' && $org ne '' && $org ne 'null') {
 		my %table = ();
-		open ORGTABLE, "$homeDir/data/knownTFs/organism.table.txt" or die "Where is the file?\n";
+		open ORGTABLE, $ENV{'HOMER_DATA'} . "/data/knownTFs/organism.table.txt" or die "Where is the file?\n";
 		while (<ORGTABLE>) {
 			next if (/^#/);
 			chomp;
@@ -375,14 +375,14 @@
 	} elsif ($mset eq 'auto') {
 		$mset='all';
 	}
-	my $msetDir = $homeDir . "/data/knownTFs/$mset/";
+	my $msetDir = $ENV{'HOMER_DATA'} . "/data/knownTFs/$mset/";
 	my $allMotifsFile = $msetDir . "all.motifs";
 	my $knownMotifsFile = $msetDir . "known.motifs";
 	unless (-e $allMotifsFile) {
 		print STDERR "\tWarning, couldn't find $allMotifsFile for mset: \"$mset\"\n";
 		print STDERR "\t\tUsing all motifs...\n";
 		$mset = 'all';
-		$msetDir = $homeDir . "/data/knownTFs/$mset/";
+		$msetDir = $ENV{'HOMER_DATA'} . "/data/knownTFs/$mset/";
 		$allMotifsFile = $msetDir . "all.motifs";
 		$knownMotifsFile = $msetDir . "known.motifs";
 	}
diff -ru --new-file homer-4.9.1.orig/bin/loadGenome.pl homer-4.9.1/bin/loadGenome.pl
--- homer-4.9.1.orig/bin/loadGenome.pl	2016-07-25 18:01:47.000000000 -0500
+++ homer-4.9.1/bin/loadGenome.pl	2017-11-07 15:54:22.557407342 -0600
@@ -22,7 +22,7 @@
 use POSIX;
 use HomerConfig;
 
-my $defaultdirectory = $homeDir . "/data/genomes/";
+my $defaultdirectory = $ENV{'HOMER_DATA'} . "/data/genomes/";
 my $config = HomerConfig::loadConfigFile();
 
 sub printCMD {
diff -ru --new-file homer-4.9.1.orig/bin/loadPromoters.pl homer-4.9.1/bin/loadPromoters.pl
--- homer-4.9.1.orig/bin/loadPromoters.pl	2016-07-05 13:15:17.000000000 -0500
+++ homer-4.9.1/bin/loadPromoters.pl	2017-11-07 15:54:34.939438052 -0600
@@ -20,7 +20,7 @@
 use POSIX;
 use HomerConfig;
 
-my $defaultdirectory = $homeDir . "/data/promoters/";
+my $defaultdirectory = $ENV{'HOMER_DATA'} . "/data/promoters/";
 my $defaultSize = 4000;
 my $fastaFlag = 0;
 my $defaultStart = -300;
diff -ru --new-file homer-4.9.1.orig/bin/makeBigBedMotifTrack.pl homer-4.9.1/bin/makeBigBedMotifTrack.pl
--- homer-4.9.1.orig/bin/makeBigBedMotifTrack.pl	2014-03-27 18:49:32.000000000 -0500
+++ homer-4.9.1/bin/makeBigBedMotifTrack.pl	2017-11-07 15:54:49.201473417 -0600
@@ -51,7 +51,7 @@
 
 `sort -k1,1 -k2,2n $tmpfile2 > $tmpfile`;
 
-my $gdir = "$homeDir" . "/data/genomes/$genome/";
+my $gdir = $ENV{'HOMER_DATA'} . "/data/genomes/$genome/";
 `homerTools extract stats "$gdir" > $tmpfile2`;
 
 open IN, $tmpfile2;
diff -ru --new-file homer-4.9.1.orig/bin/makeBigWig.pl homer-4.9.1/bin/makeBigWig.pl
--- homer-4.9.1.orig/bin/makeBigWig.pl	2017-01-17 12:56:04.000000000 -0600
+++ homer-4.9.1/bin/makeBigWig.pl	2017-11-07 15:55:28.758571473 -0600
@@ -132,7 +132,7 @@
 my $tmpFile = $rand . ".tmp";
 
 if ($chromSizeFile eq '') {
-	$chromSizeFile = $homeDir . "/data/genomes/$genome/chrom.sizes";
+	$chromSizeFile = $ENV{'HOMER_DATA'} . "/data/genomes/$genome/chrom.sizes";
 	if (-d $chromSizeFile && $forceCalc == 0) {
 	} else {
 		my $customGenomeFlag = 0;
@@ -149,7 +149,7 @@
 				#print STDERR "`homerTools extract stats \"$genome\" > $tmpFile`;\n";
 				`homerTools extract stats \"$genome\" > "$tmpFile"`;
 			} else {
-				`homerTools extract stats \"$homeDir/data/genomes/$genome/\" > "$tmpFile"`;
+				`homerTools extract stats \"$ENV{'HOMER_DATA'}/data/genomes/$genome/\" > "$tmpFile"`;
 			}
 			open IN, $tmpFile;
 			open OUT, ">$chromSizeFile";
diff -ru --new-file homer-4.9.1.orig/bin/makeMultiWigHub.pl homer-4.9.1/bin/makeMultiWigHub.pl
--- homer-4.9.1.orig/bin/makeMultiWigHub.pl	2017-01-17 12:56:23.000000000 -0600
+++ homer-4.9.1/bin/makeMultiWigHub.pl	2017-11-07 15:55:45.467612876 -0600
@@ -151,7 +151,7 @@
 }
 
 my $customGenomeFlag = 0;
-if (-d $homeDir . "/data/genomes/$genome/") {
+if (-d $ENV{'HOMER_DATA'} . "/data/genomes/$genome/") {
 } elsif (-f $genome) {
 	$customGenomeFlag = 1;
 	print STDERR "Using custom genome (i.e. fasta file)\n";
@@ -162,7 +162,7 @@
 my $tmpFile = $rand . ".tmp";
 
 if ($chromSizeFile eq '') {
-    $chromSizeFile = $homeDir . "/data/genomes/$genome/chrom.sizes";
+    $chromSizeFile = $ENV{'HOMER_DATA'} . "/data/genomes/$genome/chrom.sizes";
 	if ($customGenomeFlag) {
 		$chromSizeFile = "$genome.chrom.sizes";
 	}
@@ -174,7 +174,7 @@
 			#print STDERR "`homerTools extract stats \"$genome\" > $tmpFile`;\n";
 			`homerTools extract stats \"$genome\" > "$tmpFile"`;
 		} else {
-			`homerTools extract stats \"$homeDir/data/genomes/$genome/\" > "$tmpFile"`;
+			`homerTools extract stats \"$ENV{'HOMER_DATA'}/data/genomes/$genome/\" > "$tmpFile"`;
 		}
 		open IN, $tmpFile;
 		open OUT, ">$chromSizeFile";
diff -ru --new-file homer-4.9.1.orig/bin/preparseGenome.pl homer-4.9.1/bin/preparseGenome.pl
--- homer-4.9.1.orig/bin/preparseGenome.pl	2016-07-05 13:15:17.000000000 -0500
+++ homer-4.9.1/bin/preparseGenome.pl	2017-11-07 15:55:55.355639612 -0600
@@ -30,7 +30,7 @@
 sub cmdLineOptions {
 	print STDERR "\n\tProgram preparse the genome for use with motif finding.\n";
 	print STDERR "\n\tUsage: preparseGenome.pl <genome> -size <#> [additional options]\n";
-	print STDERR "\tOutput files will be placed in $homeDir/data/genomes/[genome]/preparsed/\n";
+	print STDERR "\tOutput files will be placed in $ENV{'HOMER_DATA'}/data/genomes/[genome]/preparsed/\n";
 	print STDERR "\t\t*.###.seq *.###.pos *.###.cgfreq *.###.cgbins *.###.gcbins\n";
 	print STDERR "\tFirst argument must be <genome>\n";
 
diff -ru --new-file homer-4.9.1.orig/bin/removeOutOfBoundsReads.pl homer-4.9.1/bin/removeOutOfBoundsReads.pl
--- homer-4.9.1.orig/bin/removeOutOfBoundsReads.pl	2016-07-05 13:15:17.000000000 -0500
+++ homer-4.9.1/bin/removeOutOfBoundsReads.pl	2017-11-07 15:56:07.584675764 -0600
@@ -69,12 +69,12 @@
 my $tmpFile = $rand . ".tmp";
 
 if ($chromSizeFile eq '') {
-	$chromSizeFile = $homeDir . "/data/genomes/$genome/chrom.sizes";
+	$chromSizeFile = $ENV{'HOMER_DATA'} . "/data/genomes/$genome/chrom.sizes";
 	if (-e $chromSizeFile && $forceCalc == 0) {
 	} else {
 		print STDERR "\t!! Could not find chromosome size file ($chromSizeFile)\n";
 		print STDERR "\t   Generating one from genome sequence files...\n";
-		`homerTools extract stats \"$homeDir/data/genomes/$genome/\" > "$tmpFile"`;
+		`homerTools extract stats \"$ENV{'HOMER_DATA'}/data/genomes/$genome/\" > "$tmpFile"`;
 		open IN, $tmpFile;
 		open OUT, ">$chromSizeFile";
 		while (<IN>) {
diff -ru --new-file homer-4.9.1.orig/configureHomer.pl homer-4.9.1/configureHomer.pl
--- homer-4.9.1.orig/configureHomer.pl	2017-02-22 18:46:16.000000000 -0600
+++ homer-4.9.1/configureHomer.pl	2017-11-07 15:43:00.798708222 -0600
@@ -155,15 +155,15 @@
 	my $url = $baseURL . "homer.misc.v1.0.zip";
 	`wget -O homer.package.zip "$url"`;
 	print STDERR "\t\tUnzipping...\n";
-	`unzip -o -d "$homeDir" homer.package.zip`;
+	`unzip -o -d "$ENV{'HOMER_DATA'}" homer.package.zip`;
 	`rm homer.package.zip`;
 }
-my $configFile = $homeDir . "/" . "config.txt";
+my $configFile = $ENV{'HOMER_DATA'} . "/" . "config.txt";
 
 if ($listFlag || $updateFlag || $removeFlag || $installFlag) { 
 
 	my $updateURL = $baseURL . "update.txt";
-	my $updateFile = $homeDir . "/" . "update.txt";
+	my $updateFile = $ENV{'HOMER_DATA'} . "/" . "update.txt";
 	`wget -O "$updateFile" "$updateURL"`;
 	my $config = loadConfigFile($configFile);
 	updateSettings($config,\%newSettings);
@@ -242,7 +242,7 @@
 				print STDERR "\t\t$g\n";
 				foreach(keys %{$removeList{$g}}) {
 					my $pkg = $_;
-					my $loc = $homeDir . $config->{$g}->{$pkg}->{'location'};
+					my $loc = $ENV{'HOMER_DATA'} . $config->{$g}->{$pkg}->{'location'};
 					my $cmd = "";
 					if ($g eq 'ORGANISMS') {
 						$cmd = "rm \"$loc/$_\"* \"$loc/../GO/$_.\"*";
@@ -267,7 +267,7 @@
 				my $g = $_;
 				foreach(keys %{$removeList{$g}}) {
 					my $pkg = $_;
-					my $loc = $homeDir . $config->{$g}->{$pkg}->{'location'};
+					my $loc = $ENV{'HOMER_DIR'} . $config->{$g}->{$pkg}->{'location'};
 					my $cmd = "";
 					if ($g eq 'ORGANISMS') {
 						$cmd = "rm \"$loc/$_\"* \"$loc/../GO/$_.\"*";
@@ -285,7 +285,7 @@
 				}
 			}
 		}
-		printConfigFile($config, $homeDir . "/config.txt");
+		printConfigFile($config, $ENV{'HOMER_DATA'} . "/config.txt");
 	}
 
 
@@ -353,7 +353,7 @@
 	}
 
 	if ($installFlag) {
-		updateConfigureScript($baseURL . "configureHomer.pl", $0 , $updateScriptFlag, @ARGV);
+		#updateConfigureScript($baseURL . "configureHomer.pl", $0 , $updateScriptFlag, @ARGV);
 
 		my @depCheck = ("PROMOTERS","GENOMES");
 		foreach(@depCheck) {
@@ -400,22 +400,22 @@
 				print STDERR "\t\tDownloading...\n";
 				`wget -O homer.package.zip "$url"`;
 				print STDERR "\t\tUnzipping...\n";
-				`unzip -o -d "$homeDir" homer.package.zip`;
+				`unzip -o -d "$ENV{'HOMER_DATA'}" homer.package.zip`;
 				`rm homer.package.zip`;
 				$config->{$mode}->{$package} = $update->{$mode}->{$package};
 				print STDERR "\t\tFinished Installing $package\n\n";
 			}
 		}
 		if (exists($install{'homer'}) || $installList{'SOFTWARE'}->{'homer'}) {
-			compileSoftware();
+			#compileSoftware();
 		}
-		printConfigFile($config, $homeDir . "/config.txt");
+		printConfigFile($config, $ENV{'HOMER_DATA'}  . "/config.txt");
 	}
 } elsif (scalar(keys %newSettings) > 0) {
 
 	my $config = loadConfigFile($configFile);
 	updateSettings($config,\%newSettings);
-	printConfigFile($config, $homeDir . "/config.txt");
+	printConfigFile($config, $ENV{'HOMER_DATA'} . "/config.txt");
 
 }
 exit;
diff -ru --new-file homer-4.9.1.orig/.ls homer-4.9.1/.ls
--- homer-4.9.1.orig/.ls	1969-12-31 18:00:00.000000000 -0600
+++ homer-4.9.1/.ls	2017-11-07 14:32:55.615817256 -0600
@@ -0,0 +1 @@
+/home/a-m/easybuild/temp/homer/homer-4.9.1
diff -ru --new-file homer-4.9.1.orig/Makefile homer-4.9.1/Makefile
--- homer-4.9.1.orig/Makefile	1969-12-31 18:00:00.000000000 -0600
+++ homer-4.9.1/Makefile	2017-10-17 10:59:48.884114222 -0500
@@ -0,0 +1,10 @@
+SUBDIR = cpp
+
+all:
+	$(MAKE) -C $(SUBDIR)
+
+clean:
+	$(MAKE) -C $(SUBDIR) clean
+
+
+.PHONY: all $(SUBDIRS)
