diff -ru GenomicConsensus-smrtanalysis-3.0.2.orig/GenomicConsensus/options.py GenomicConsensus-smrtanalysis-3.0.2/GenomicConsensus/options.py
--- GenomicConsensus-smrtanalysis-3.0.2.orig/GenomicConsensus/options.py	2018-02-21 11:11:38.197243476 -0600
+++ GenomicConsensus-smrtanalysis-3.0.2/GenomicConsensus/options.py	2018-02-21 11:11:58.283291964 -0600
@@ -298,16 +298,16 @@
              "which selects the best parameter set from the cmp.h5")
 
     debugging = parser.add_argument_group("Verbosity and debugging/profiling")
-    debugging.add_argument(
-        "--verbose",
-        dest="verbosity",
-        action="count",
-        help="Set the verbosity level.")
-    debugging.add_argument(
-        "--quiet",
-        dest="quiet",
-        action="store_true",
-        help="Turn off all logging, including warnings")
+#    debugging.add_argument(
+#        "--verbose",
+#        dest="verbosity",
+#        action="count",
+#        help="Set the verbosity level.")
+#    debugging.add_argument(
+#        "--quiet",
+#        dest="quiet",
+#        action="store_true",
+#        help="Turn off all logging, including warnings")
     debugging.add_argument(
         "--profile",
         action="store_true",
