diff -ru prokka-1.13.orig/bin/prokka prokka-1.13/bin/prokka
--- prokka-1.13.orig/bin/prokka	2018-08-28 09:40:19.717863719 -0500
+++ prokka-1.13/bin/prokka	2018-08-28 09:47:02.287978486 -0500
@@ -20,7 +20,7 @@
 use strict;
 use warnings;
 use FindBin;
-use lib "$FindBin::RealBin/../perl5"; # for bundled Perl modules
+#use lib "$FindBin::RealBin/../perl5"; # for bundled Perl modules
 use File::Copy;
 use Time::Piece;
 use Time::Seconds;
@@ -570,7 +570,7 @@
     msg("Running RNAmmer");
     my $rnammerfn = "$outdir/rnammer.xml";
     my $num_rrna = 0;
-    my $rnammer_opt = $cpus != 1 ? "-multi" : "";
+    my $rnammer_opt = $cpus != 1 ? "-multi" : " -T /scratch ";
     runcmd("rnammer -S $rnammer_mode $rnammer_opt -xml \Q$rnammerfn\E \Q$outdir/$prefix.fna\E");
     my $xml = XML::Simple->new(ForceArray => 1);
     my $data = $xml->XMLin($rnammerfn);
