diff -ru diamond-0.9.1.original/CMakeLists.txt diamond-0.9.1/CMakeLists.txt
--- diamond-0.9.1.original/CMakeLists.txt	2017-09-25 14:06:28.349869088 -0500
+++ diamond-0.9.1/CMakeLists.txt	2017-09-25 14:06:43.246912921 -0500
@@ -10,9 +10,9 @@
 endif()
 
 include(CheckCXXCompilerFlag)
-CHECK_CXX_COMPILER_FLAG("-march=native" COMPILER_SUPPORTS_MARCHNATIVE)
+CHECK_CXX_COMPILER_FLAG("-march=generic" COMPILER_SUPPORTS_MARCHNATIVE)
 if(COMPILER_SUPPORTS_MARCHNATIVE)
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -march=native")
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -march=generic")
 endif()
 
 find_package(ZLIB REQUIRED)
