diff -ru CNVnator-de012f2bccfd4e11e84cf685b19fc138115f2d0d.orig/Makefile CNVnator-de012f2bccfd4e11e84cf685b19fc138115f2d0d/Makefile
--- CNVnator-de012f2bccfd4e11e84cf685b19fc138115f2d0d.orig/Makefile	2018-11-27 16:34:58.703485000 -0600
+++ CNVnator-de012f2bccfd4e11e84cf685b19fc138115f2d0d/Makefile	2018-11-27 16:36:14.626102000 -0600
@@ -1,5 +1,6 @@
 VERSION	       = v0.3.3
 override LIBS += -lz -lbz2 -lcurl -llzma
+HTSDIR = $(EBROOTHTSLIB)
 
 ifneq ($(wildcard $(ROOTSYS)/lib/root),)
         ROOTLIBS = -L$(ROOTSYS)/lib/root -lCore -lRIO -lHist -lGraf -lGpad -lTree -lMathCore
@@ -8,17 +9,14 @@
 endif
 
 ifneq ($(wildcard $(ROOTSYS)/include/root),)
-        INC = -I$(ROOTSYS)/include/root -I$(SAMDIR)
+        INC = -I$(ROOTSYS)/include/root -I$(HTSDIR)/include/htslib
 else
-        INC = -I$(ROOTSYS)/include      -I$(SAMDIR)
+        INC = -I$(ROOTSYS)/include      -I$(HTSDIR)/include/htslib
 endif
 
-SAMDIR = samtools
-SAMLIB = $(SAMDIR)/libbam.a
-HTSDIR = $(wildcard $(SAMDIR)/htslib-*)
 ifneq ($(HTSDIR),)
-        SAMLIB += $(HTSDIR)/libhts.a
-        INC    += -I$(HTSDIR)
+        SAMLIB += $(HTSDIR)/lib/libhts.a
+        INC    += -I$(HTSDIR)/include/htslib
 endif
 
 ifeq ($(OMP),no)
