diff -ru IGV_2.8.0.orig/igv.sh IGV_2.8.0/igv.sh
--- IGV_2.8.0.orig/igv.sh	2020-12-11 07:56:09.822559000 -0600
+++ IGV_2.8.0/igv.sh	2020-12-11 07:56:26.614027000 -0600
@@ -16,7 +16,7 @@
     echo "Using system JDK."
 fi
 
-exec java -showversion --module-path="${prefix}/lib" -Xmx4g \
+exec java -showversion --module-path="${prefix}/lib" -Xmx15g \
     @"${prefix}/igv.args" \
     -Dapple.laf.useScreenMenuBar=true \
     -Djava.net.preferIPv4Stack=true \
