diff -ru MaxBin-2.2.7.orig/_getabund.pl MaxBin-2.2.7/_getabund.pl
--- MaxBin-2.2.7.orig/_getabund.pl	2020-05-18 11:01:44.761778000 -0500
+++ MaxBin-2.2.7/_getabund.pl	2020-05-18 11:02:13.524328000 -0500
@@ -1,4 +1,4 @@
-#!/usr/bin/perl -w
+#!/usr/bin/env perl
 use strict;
 
 #getsam($ARGV[0], $ARGV[1]);
diff -ru MaxBin-2.2.7.orig/_getmarker.pl MaxBin-2.2.7/_getmarker.pl
--- MaxBin-2.2.7.orig/_getmarker.pl	2020-05-18 11:01:44.762789000 -0500
+++ MaxBin-2.2.7/_getmarker.pl	2020-05-18 11:02:20.676838828 -0500
@@ -1,4 +1,4 @@
-#!/usr/bin/perl -w
+#!/usr/bin/env perl
 use strict;
 
 my $COV_CUTOFF = 0.4;
diff -ru MaxBin-2.2.7.orig/run_MaxBin.pl MaxBin-2.2.7/run_MaxBin.pl
--- MaxBin-2.2.7.orig/run_MaxBin.pl	2020-05-18 11:01:44.761042000 -0500
+++ MaxBin-2.2.7/run_MaxBin.pl	2020-05-18 11:02:32.614893561 -0500
@@ -1,4 +1,4 @@
-#!/usr/bin/perl -w
+#!/usr/bin/env perl
 use strict;
 use Cwd;
 use LWP::Simple;
