diff -ru kneaddata-0.6.1.orig/kneaddata/utilities.py kneaddata-0.6.1/kneaddata/utilities.py
--- kneaddata-0.6.1.orig/kneaddata/utilities.py	2018-11-02 16:19:55.417634000 -0500
+++ kneaddata-0.6.1/kneaddata/utilities.py	2018-11-02 16:33:51.329503836 -0500
@@ -243,19 +243,21 @@
     message="Reformatting file sequence identifiers ..."
     print(message+"\n")
     logger.info(message)   
-    
+   
     file_out, new_file=tempfile.mkstemp(prefix="reformatted_identifiers",
         suffix="_"+file_without_extension(file), dir=output_folder)
-    
-    for lines in read_file_n_lines(file,4):
+
+    with open(new_file, "w") as outfile:
+        for lines in read_file_n_lines(file,4):
         # reformat the identifier and write to temp file
-        if " 1:" in lines[0]:
-            lines[0]=lines[0].replace(" 1","").rstrip()+"#0/1\n"
-        elif " 2:" in lines[0]:
-            lines[0]=lines[0].replace(" 2","").rstrip()+"#0/2\n"
-        else:
-            lines[0]=lines[0].replace(" ","")
-        os.write(file_out, "".join(lines))
+            if " 1:" in lines[0]:
+                lines[0]=lines[0].replace(" 1","").rstrip()+"#0/1\n"
+            elif " 2:" in lines[0]:
+                lines[0]=lines[0].replace(" 2","").rstrip()+"#0/2\n"
+            else:
+                lines[0]=lines[0].replace(" ","")
+            outfile.write("".join(lines))
+ 
         
     os.close(file_out)
     
